### ISSUE TITLE
I like this! 

### ISSUE THREAD

**Issue by linwaytin** [cjdoris/Chevy.jl#1]
Julia really needs a better piping mechanism. I like the `>>` notation together with `_` representing the input. 

I noticed that we can use `enable_repl` to save the macro call. Is it possible to use this feature in a script, or a module?

Thank you.

**Comment by cjdoris** [comment]
Glad you like it :) Maybe I'll register it...

If you want to use this in a script or module, you will need to use the `@chevy` macro, such as

```
@chevy module
    # your module...
end
```

or

```
@chevy begin
    # your script...
end
```

**Comment by linwaytin** [comment]
Cool! Do it work on functions like this?
```julia
@chevy function foo()
        # some code
end
```

**Comment by cjdoris** [comment]
Yep you can use `@chevy` on any code and it applies to any `>>` in the code, no matter how deeply nested.

**Comment by linwaytin** [comment]
Awesome!
